### PR TITLE
Changes Skeleton movespeed to be equal to humans

### PR DESCRIPTION
**Pull request by Megamaw**
Hey, guys, you know what'd be really spooky this Halloween?

**_Megamaw touching skeleton code again._**

Nah, don't worry. I'm reducing their speed to be identical to humans.

Happy Halloween!
:cl:
 * tweak: Skeletons are now as fast as humans.

### CODE REVIEW

**Comment by Exxion** [comment]
You *really* want those roundstart skelememes back, don't you?

**Comment by Shadowmech88** [comment]
Skeletons have been roundstart all of October, right?
How have players handled that so far, even with the increased speed?

**Comment by Megamaw** [comment]
Exxion, i'm more than fine with them being just a Halloween thing. It's been a few months, and I've grown to accept that.

I can't say I've been super active lately, but looking back, 150% human speed doesn't seem as fair as I thought. 

I figured it was good timing on my part to do this now since skeletons were available to the public right now.

I personally believe whether or not it's available roundstart shouldn't affect the fact that we should probably reconsider how we're balancing skeletons. I did a lot of my balancing based on how I remembered the game back in ~10-11, before things like popped lungs and other nuisances were frequent. Which was very much a mistake that I'd like to remedy.

I'm not afraid to admit I made a lotta mistakes back then.

If you remove all the subtext, strip away all the thoughts you have about 'roundstart skeletons'-- because mind you, we already have them right now-- what's your issue with making them more in line with humans speedwise?

**Comment by DamianX** [comment]
Haha, this has NOTHING to do with roundstart skellingtons, how silly of you guys to think such a thing :)

**Comment by ChugMonkey** [comment]
Man, your the reason skeletons were removed round start in the first place. Stop fucking with skeletons, for the LOVE OF GOD. I just want skeletons to remain untouched, if you honestly god damn care about skeletons, go and stay go. You're like a cancer that keeps coming back, JUST LEAVE AND NEVER RETURN IT WOULD BE A GOOD THING

t. Skeleton Poster

File: code/modules/mob/living/carbon/species.dm [hunk 1]
```diff
@@ -337,7 +337,7 @@ var/global/list/whitelisted_species = list("Human")
 		"brain" =    /datum/organ/internal/brain,
 		)
 
-	move_speed_multiplier = 1.5
+	move_speed_multiplier = 1
```
Comment (Shadowmech88) [inline code/modules/mob/living/carbon/species.dm:340]: You can actually just remove this line altogether. The default speed of `/datum/species` is 1, so all you have to do is not override it.

**Comment by N3X15** [comment]
Justify your changes in the description.  I want to know why you think this change is necessary.

**Comment by yclatious** [comment]
This ain't funny, this ain't spooky, this is megamaw being desperate to play Rattle Boners as roundstart again or just to buff skeletons for no good reason as per his last PR, in which we called out on his non existent argument for the actual validity of skelememes getting buffed in any way, shape or form. He spews deflector talk, says he did mistakes, wants everyone to forget how retarded it all was, how he ignored everyone calling out on the buffs making no logical sense due to skeletons not being anything but jokes and asks people to forget about every issue about roundstart and ask the following question, what's the issue with making skeletons as quick as humans?

My answer is obvious, its a clear bullshiting way to get skelememes into roundstart, but even if I ignore that, skeletons, roundstart or not, are and have always been a joke race for halloween and little but that, they rattle the bones in their chest to play the xylophone, ooh spoooky, they say ack ack a lot, they eat raw meat, it has always been a joke race, one that never deserved any buffs but now that the superior speed buff is in, it might aswell stay in, not only because of it being a bullshit excuse to bring em back but they now actually have atleast one small difference that makes them more than just a pure joke about skeletons and brittle bones. He essentially wants to remove the only non memetic thing about skeletons.

**Comment by Diavill** [comment]
the ride never ends ACK ACK

**Comment by SysadminJeroen** [comment]
Crack the ack.

Op 27 okt. 2017 12:22 schreef "Diavill" <notifications@github.com>:

**Comment by SonixApache** [comment]
fuck off with your skelecreep

**Comment by Shadowmech88** [comment]
>just to buff skeletons for no good reason
>skelecreep

I feel like there may be a misperception here.
This PR is strictly a nerf to skeletons as they are now.

**Comment by SonixApache** [comment]
phase 1: skeletons as fast as humans
phase 2: some other shitty minor change
phase x: roundstart skeletons

it's megameme's skelecreep plan of world bonination

**Comment by Riekland** [comment]
isn't this literally the exact same as what happened the last two times
why is megamaw allowed to speak

**Comment by PJB3005** [comment]
On one side I want to say "eh it's Halloween".

On the other I'm saying "yeah no you're not getting skeletons ever again"

**Comment by Shadowmech88** [comment]
@PJB3005 
>yeah no you're not getting skeletons ever again

Skeletons are currently selectable at round start.

**Comment by Exxion** [comment]
For the Halloween season only

**Comment by Shadowmech88** [comment]
I know, but I wasn't sure from PJB's post whether he knew they were available at all.

**Comment by PJB3005** [comment]
Wait they were re-enabled because of Halloween?
OK that's already more than enough favor can I close this?

**Comment by yclatious** [comment]
Cease this meme and be a good lad @PJB3005

**Pull request closed without merging**